JSDemos/Demos/Scheduler/RecurringAppointments/description.md

```diff
@@ -1 +1,19 @@
-This example demonstrates how to&nbsp;define recurring appointments. You can specify the appointment repetition interval programmatically or&nbsp;in&nbsp;the UI. In&nbsp;the former case, the appointment object should contain the **recurrenceRule** field. In&nbsp;the latter case, toggle the **Repeat** switcher on&nbsp;the appointment details form to&nbsp;mark the appointment as&nbsp;recurring and specify the required fields.
\ No newline at end of file
+A recurring appointment repeats at a specified interval. You can create such an appointment in the UI or programmatically.
+
+To create a recurring appointment programmatically, do the following:
+
+1. Locate or create only a single appointment object in the [dataSource](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#dataSource). 
```

Comment:
Could you please explain this sentence? I see several objects in the data source 
![image](https://user-images.githubusercontent.com/33040934/117450223-9dcf1600-af49-11eb-856e-cbb98e28ef3b.png)